_resources/production_liasion_guidelines.md

```diff
@@ -0,0 +1,125 @@
+
+Mention in the ticket that you will be submitting simulation jobs to OSG on behalf of the ePIC experiment. Once approved, you will be able to login to osg-eic after logging into the JLab login node. 
+
+```bash
+ssh <username>@osg-eic -Y
+# Use your 2-factor Safenet password
+```
+
+If you are having trouble with account access, call JeffersonLab helpdesk at +1(757)2697155.
+
+### Open Science Grid
+```bash
+ssh <username>@ap23.uc.osg-htc.org -Y
+# If you already set up your SSH keys on ci-connect, you will only need to enter your 2-factor password
+```
+
+If you are having trouble with account access, email support@osg-htc.org.
+
+### Brookhaven National Lab
+```bash
+ssh <username>@ssh.sdcc.bnl.gov -Y
+```
+
+Once you have access to the sdcc login node, file a ticket with RT-RACF-UserAccounts@bnl.gov to get access to the osgsub01 node and mention that that you will be using it to submit jobs on behalf of the ePIC collaboration. Once approved, you can log onto the access point after logging onto the sdcc login node.  
+
+```bash
+ssh <username>@osgsub01 -Y
+```
+
+Report any difficulties to the above-mentioned email for filing tickets. 
+
+---
+# Getting Write Access to Jefferson Lab Rucio Storage Endpoint (RSE)
+### Getting a Certificate and Key
+
+1. If you have a jlab account, file a ticket with [Service Now](https://jlab.servicenowservices.com/) asking for a certificate to be able access ePIC Rucio in the context of OSG job submission. If you don't have a jlab account, contact the [production WG](https://eic.github.io/epic-prod/about/contact.html) to file a ticket on your behalf.
+
+2. You will receive an email from support@cert-manager.com with the instruction to download your .cer file.
+​
+3. Once you have received this .cer file, transfer this to the access point you will be using. Please request the production WG to provide you with the key corresponding to your .cer file. Your key will be placed in the access point according to your request (BNL, JLAB or OSG). 
+​
+4. Move certificates and keys to a desired location and change the permissions.
+    ```bash
+    chmod 600 usercert.cer
+    chmod 600 userkey.key
+    ```
+
+### Installing voms-client
+
+You will need the `voms-client` on a Linux machine. If you are on your local Linux machine, then do:
+
+```bash
+sudo apt-get install voms-clients-java
+```
+All the access points should already have the `voms-client` installed. So, you can skip this step if you are already on one. 
+
+### Running voms-proxy-init
+
+Run the following command to generate a voms proxy certificate with a validity of at least 2 months (1460 hours):
+
+```bash
+voms-proxy-init -cert=usercert.cer -key=userkey.key -out=x509_user_proxy -hours=1460
+```
+
+This will create an proxy certificate in your current directory. 
+
+### Send Information
+
+Run the following command and email the output to `panta@jlab.org`:
```

Comment:
Can we get a workflow for this that doesn't depend on a single person but is routed through service now?

Reply:
create a incident at Jlab service now.
(this requires Jlab account)